packages/amazonq/.changes/next-release/Feature-06bf59c2-da3f-4ab5-ab86-634b3c25bd5e.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "send metric data in onCodeGeneration"
+}
```

Comment:
This happens in the background and the user doesn't notice. We don't need a changelog item.
```suggestion
```